x/storage/client/cli/tx_post_file.go

```diff
@@ -230,12 +230,13 @@ func postFile(fileData []byte, cmd *cobra.Command) {
 		panic(err)
 	}
 
+	if res != nil {
+		fmt.Println(res.RawLog)
```

Comment:
is this debugging code?

Reply:
Kind of yeah, it's mildly helpful though always

Reply:
Generally the cli in this instance should really only be for debugging anyways